### ISSUE TITLE
cant use code action

### ISSUE THREAD

**Issue by Oveln** [nvimdev/lspsaga.nvim#1405]
### Describe the bug

when i use code action by lspsaga, it appear a strange problem
This may be caused by the update of the nvim version ?

The following is an error report：```

Error detected while processing CursorMoved Autocommands for "<buffer=5>":
Error executing lua callback: ...vim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/preview.lua:99: attempt to index field 'col' (a number value)
stack traceback:
        ...vim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/preview.lua:99: in function 'create_preview_win'
        ...vim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/preview.lua:152: in function 'action_preview'
        ...e/nvim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/init.lua:239: in function 'set_cursor'
        ...e/nvim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/init.lua:99: in function <...e/nvim/lazy/lspsaga.nvim/lua/lspsaga/codeaction/init
.lua:98>

```

### Steps to reproduce

execute :Lspsaga code_action

### Expected behavior

Normal display

### Neovim version (nvim -v)

NVIM v0.10.0-dev

### lspsaga commit

7cabe94

### Terminal name/version

tmux 3.4

**Comment by lervag** [comment]
I think I've fixed this in #1406 - I hope a maintainer can review and merge the PR.

**Comment by lervag** [comment]
I believe this was fixed in the latest patch.
